.changeset/fifty-pans-fail.md

```diff
@@ -0,0 +1,5 @@
+---
+'@firebase/vertexai': patch
```

Comment:
Should be "minor" as we change the public API. Also add "firebase: minor"

Reply:
done, I created a new changeset